tools/topology/sof/pipe-simple-playback.m4

```diff
@@ -0,0 +1,26 @@
+# Low Latency Passthrough Pipeline and PCM
```

Comment:
this comment is wrong

Reply:
besides this is hardly a pipeline with the SectionGraph defined elsewhere. All you have is a buffer?

Reply:
> besides this is hardly a pipeline with the SectionGraph defined elsewhere. All you have is a buffer?

I will have a buffer with a pipeline id which is required for connecting to a demux.